README.md

```diff
@@ -717,32 +755,34 @@ Shuts down any pending operations on the socket, and cleans up state. The socket
 
     int aws_socket_connect(struct aws_socket *socket, struct aws_socket_endpoint *remote_endpoint);
 
-Connects to a remote endpoint. In UDP, this simply binds the socket to a remote address for use with `aws_socket_write()`,
-and if the operation is successful, the socket can immediately be used for write operations.
+Connects to a remote endpoint. In TCP and all Apple Network Framework connections (regardless it is UDP, TCP or LOCAL), this will 
```

Comment:
"this will function will not block." reword

Also, I don't know if this is the right wording.  Saying "will not block" is misleading; none of them "block" in the sense that they're waiting on some async process to complete, it's just sometimes the socket is immediately usable and sometimes it isn't.